packages/generator/tests/test_assets/example_renders/measurement_plugin_client/non_streaming_data_measurement_client.py

```diff
@@ -298,27 +305,70 @@ def __init__(
         if grpc_channel is not None:
             self._stub = v2_measurement_service_pb2_grpc.MeasurementServiceStub(grpc_channel)
         self._create_file_descriptor()
+        self._pin_map_context: Optional[PinMapContext] = None
+
+    @property
+    def pin_map_context(self) -> PinMapContext:
+        """Get the pin map context for the RPC."""
+        return self._pin_map_context
+
+    @pin_map_context.setter
+    def pin_map_context(self, val: PinMapContext) -> None:
+        if not isinstance(val, PinMapContext):
+            raise ValueError("pin_map_context must be an instance of PinMapContext.")
```

Comment:
Use TypeError.

Include the invalid type in the error message.

See https://codesearch.natinst.com/search/text?q=raise%20TypeError%20repo%3Agithub-meas&fold_case=auto&regex=true&context=true for examples.

Reply:
Updated the error message.

Reply:
Please include the invalid type in the error message.